package.json

```diff
@@ -28,39 +28,54 @@
   "bugs": {
     "url": "https://github.com/aerospike/aerospike-client-nodejs/issues"
   },
+  "binary": {
+    "module_name": "aerospike",
+    "module_path": "./lib/binding/{configuration}/{node_abi}-{platform}-{arch}/",
+    "remote_path": "./{module_name}/v{version}/{configuration}/",
+    "package_name": "{module_name}-v{version}-{node_abi}-{platform}-{arch}.tar.gz",
+    "host": "https://www.npmjs.com/package/aerospike"
+  },
+  "config": {
+    "unsafe-perm": true
+  },
```

Comment:
Is this really required? unsafe-perm defaults to true for non-root users anyway: https://docs.npmjs.com/cli/v6/using-npm/config#unsafe-perm